### ISSUE TITLE
Editor - Trigger game entitres

### ISSUE THREAD

**Issue by razovu** [ddnet/ddnet#7934]
Greetings to the DDnet team. 
I want to offer an idea of how to bring diversity to mapping :)
If possible, it would be cool to make an additional entity called "trigger". 
In fact, the DDnet client already has a similar entity called "tunezone" `tune_zone_enter i[zone] r[message]` 
What's the point? 
Tunezone has a very limited scope of application, unlike a trigger. But just imagine, we could create interactive maps and how much more creative it can be?
So we have 2 different ways. 
**1st one - add more server commands like** 
```
animation i[start/end] x[animation's number]
animation start 1
sound start 1

countdown i[command] t[time (ms)]
countdown freeze 3000ms

/** it can be quadanimation, sound, moving tiles, etc */
```
**problem1** how not to shoot yourself in the foot to an inexperienced mapper?
**problem2** how to split tunezone for many ppl? U know... the server may die if someone thinks of endlessly activating the trigger. 
So that's why we need the 2nd way i guess
2nd way - Add a unique entity trigger that affects not only the person who activated it, but also all the people on the map. + it can be activated only 1 time then just self-destruct (which will be the solution to the 2nd problem)

What do you think guys?

**Comment by razovu** [comment]
Of course, I could but clearly express my thought, but I hope you have caught the essence xD

**Comment by l-ouis** [comment]
Related to #7666 (and somewhat #6683)
